### ISSUE TITLE
Add support for forms in MS Word

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8565]
### Steps to reproduce:
1. Open MS Word
2. Insert a form (i.e. a line or a rectangle or a circle)
3. Try to identify the form with the caret

### Actual behavior:
NVDA does not announce the forms at all.

### Expected behavior:
NVDA should announce the forms and allow selecting them.

#### NVDA Installed/portable/running from source:
Both

#### NVDA version:
2018.2.1.

#### Windows version:
Windows 8.1.

#### Name and version of other software in use when reproducing the issue:
MS Word 2010

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
no

**Comment by seanbudd** [comment]
@Adriani90 - is this still an issue with the latest versions of the software and NVDA?

**Comment by Adriani90** [comment]
This works as expected with UIA enabled, but not with objectModel. Tested in MS Office 365 version 2507.
